public/store/reducers/models_reducer.ts

```diff
@@ -20,7 +20,7 @@ const SEARCH_MODELS_ACTION = `${MODELS_ACTION_PREFIX}/searchModels`;
 export const searchModels = createAsyncThunk(
   SEARCH_MODELS_ACTION,
   async (
-    { apiBody, dataSourceId }: { apiBody: {}, dataSourceId: string },
+    { apiBody, dataSourceId }: { apiBody: {}; dataSourceId?: string },
```

Comment:
does this not cause errors?

Reply:
I don't think this will cause any issue.

Reply:
did it auto-correct to this? iirc there is compilation issues between using `,` and `;`. but if you're not facing issues, sounds fine

Reply:
- Yes, the formatter autocorrected. I didn't see any issues with using `;`
- I will remove this in the next PR, in case it needs to be rolled back.